### ISSUE TITLE
socketio channel in Rasa 3.0 is not working

### ISSUE THREAD

**Issue by medicareamaze** [RasaHQ/rasa#10411]
### Rasa Open Source version

rasa -3.0.0

### Rasa SDK version

3.0.0

### Python version

3.8

### What operating system are you using?

Linux, OSX

### What happened?

Installed a fresh Rasa 3.0 and tried to connect from my website using socketio endpoint. Rasa 3.0.0 throws exception

### Command / Request

```shell
Trigger a Get request to http://localhost:5005/socket.io/?EIO=4&transport=polling&t=NrceI9f
```

### Relevant log output

```shell
`Exception occurred while handling uri: ‘http://localhost:5005/socket.io/?EIO=4&transport=polling&t=NrceI9f’ Traceback (most recent call last): File “handle_request”, line 83, in handle_request class Sanic(BaseSanic, metaclass=TouchUpMeta): File “/opt/anaconda3/lib/python3.8/site-packages/engineio/asyncio_server.py”, line 317, in handle_request return await self._make_response(r, environ) File “/opt/anaconda3/lib/python3.8/site-packages/engineio/asyncio_server.py”, line 385, in _make_response response = make_response( File “/opt/anaconda3/lib/python3.8/site-packages/engineio/async_drivers/sanic.py”, line 102, in make_response return HTTPResponse(body=payload, content_type=content_type, TypeError: ‘NoneType’ object is not callable`
```

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @yennycheung will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by m-vdb** [comment]
This is a duplicate of https://github.com/RasaHQ/rasa/issues/10412 and has been fixed in 3.0.9
